doc/lua_api.txt

```diff
@@ -2278,8 +2278,16 @@ These functions return the leftover itemstack.
 
 * `minetest.forceload_free_block(pos)`
     * stops forceloading the position `pos`
-
 Please note that forceloaded areas are saved when the server restarts.
+* `minetest.register_on_punchplayer(player, hitter, time_from_last_punch, tool_capabilities, dir)`
```

Comment:
Fixed. It was in the original pull request like this and I didn't see the mistake.